### PR TITLE
UI Adjustment and enhancements

### LINKED ISSUE #5095: Small adjustments
- [x] when creating a new cluster. cluster online state and machines online state have different spinners, please unify
- [x] Error icon breaks into next line when it has no apparent reason to do that 
error breaks into next line when it has no apparent reason to

- [x] Add + button to table row for Service accounts and Backup Destinations (admin panel) (on that note a question: Can Backup Destination Seeds not be deleted? We don't have a delete option for it in the UI) 
add + to service account row

- [x] header icons not vertically centered
not vertically centered

- [x] Resource dropdown, items break line on click
on click cluster template breaks line

- [x] Auto collapse sidebar at 1200px browserwidth (Behavior: sidebar collapses automatically if not collapsed already. If the user manually uncollapses the sidebar, it's permitted and we do not enforce the collapsed style) 

- [x] OPA dialogs Admin Panel increase YAML field height 
admin panel opa increase text field size

- [x] Fix "Edit Preset Provider" Spacing (too large, not sure what's up here)
funky spacing and change confirm ux

### PR DESCRIPTION
**Pull request by KhizerRehan**
**What this PR does / why we need it**:

This PR adds UI adjustments and enhancements

**Which issue(s) this PR fixes**:

Fixes #5095 

Adding some of the screenshot

**header icons alignment**

vertiical-aligned

**Fix error icon:**

warning-icon

info-icon

**Default Loading spinner**

https://user-images.githubusercontent.com/17727069/203315282-6e7ec1eb-dd89-45b8-a9f1-83427855203c.mp4

**Add Token/Add Destination**

https://user-images.githubusercontent.com/17727069/203768726-2e652e7d-ba15-4c9f-8aa7-ed41217cdb3f.mp4

**Reducing Layout gap between children's provider presets settings**

https://user-images.githubusercontent.com/17727069/203768249-643c805d-81ca-4d99-88d1-f681489c6684.mp4

**What type of PR is this?**
/kind design
/kind cleanup
/kind chore

**Special notes for your reviewer**:

This things are already confirmed from **chiara** and has been told following changes will be reflected ALL over the application.

**Reasons**
To Stay consistent and Uniform all over the application
	- [Height of YAML Editor](https://github.com/kubermatic/dashboard/pull/5317/files#diff-f228f7c1ab6b0006015bbb2c2bfb20ed630b8b3cd6c31e8154586c8ad1124707R36)
	- [Default Loading Icon](https://github.com/kubermatic/dashboard/pull/5317/files#diff-8799641bbdf8e9c1550d1e30f30a9d7ecaf6057c0fa2e14966bedd959b39756dR24)

**Regarding this point in ticket**
-  "Auto collapse sidebar at 1200px browse width (Behavior: sidebar collapses automatically if not collapsed already...". 

This will be fixed in following ticket [Project Sidebar Redesign](https://github.com/kubermatic/dashboard/issues/5150) 
  - Sidebar is changed
  - Collapsing will take effect based on browser width so it will collapsed based on side/dimension of screen.

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by Talha-Jamil-TJ — APPROVED** [review]
/lgtm

**Comment by ahmedwaleedmalik** [comment]
Other than the bug that @ahmadhamzh mentioned, it looks good.
/retest

File: modules/web/src/assets/css/global/_main.scss [hunk 1]
```diff
@@ -533,6 +533,13 @@ km-dashboard {
       overflow-y: auto;
       padding-right: 24px;
     }
+
+    // Override material default "Forms" field padding
+    & km-preset-settings-step {
```
Comment (Waseem826) [inline modules/web/src/assets/css/global/_main.scss:538]: We're using `fxLayoutGap` property on `form` inside edit provider components for each provider so we should reduce that value instead.

```
      <km-aks-settings *ngSwitchCase="Providers.AKS"
                       [formControl]="form.get(Controls.Settings)">
      </km-aks-settings>
      <km-alibaba-settings *ngSwitchCase="Providers.ALIBABA"
                           [formControl]="form.get(Controls.Settings)">
      </km-alibaba-settings>
      <km-anexia-settings *ngSwitchCase="Providers.ANEXIA"
                          [formControl]="form.get(Controls.Settings)">
      </km-anexia-settings>
      <km-aws-settings *ngSwitchCase="Providers.AWS"
                       [formControl]="form.get(Controls.Settings)">
      </km-aws-settings>
      <km-azure-settings *ngSwitchCase="Providers.AZURE"
                         [formControl]="form.get(Controls.Settings)">
      </km-azure-settings>
      <km-digitalocean-settings *ngSwitchCase="Providers.DIGITALOCEAN"
                                [formControl]="form.get(Controls.Settings)">
      </km-digitalocean-settings>
      <km-eks-settings *ngSwitchCase="Providers.EKS"
                       [formControl]="form.get(Controls.Settings)">
      </km-eks-settings>
      <km-gcp-settings *ngSwitchCase="Providers.GCP"
                       [formControl]="form.get(Controls.Settings)">
      </km-gcp-settings>
      <km-gke-settings *ngSwitchCase="Providers.GKE"
                       [formControl]="form.get(Controls.Settings)">
      </km-gke-settings>
      <km-hetzner-settings *ngSwitchCase="Providers.HETZNER"
                           [formControl]="form.get(Controls.Settings)">
      </km-hetzner-settings>
      <km-kubevirt-settings *ngSwitchCase="Providers.KUBEVIRT"
                            [formControl]="form.get(Controls.Settings)">
      </km-kubevirt-settings>
      <km-openstack-settings *ngSwitchCase="Providers.OPENSTACK"
                             [formControl]="form.get(Controls.Settings)">
      </km-openstack-settings>
      <km-equinix-settings *ngSwitchCase="Providers.EQUINIX"
                           [formControl]="form.get(Controls.Settings)">
      </km-equinix-settings>
      <km-nutanix-settings *ngSwitchCase="Providers.NUTANIX"
                           [formControl]="form.get(Controls.Settings)">
      </km-nutanix-settings>
      <km-vsphere-settings *ngSwitchCase="Providers.VSPHERE"
                           [formControl]="form.get(Controls.Settings)">
      </km-vsphere-settings>

**Review by Waseem826 — APPROVED** [review]

**Review by Waseem826 — APPROVED** [review]

**Pull request merged**